src/libraries/LibTurnBasedGame.sol

```diff
@@ -314,7 +317,25 @@ library LibTBG {
         GameInstance storage _game = _getGame(gameId);
         if (!_game.hasStarted || isGameOver(gameId)) return false;
         if (turnTimedOut) return true;
-        return false;
+
+        /// @dev Early turn ending is possible if all active members made their move.
+        bool hasActivePlayer = false;
+
+        if (_game.currentTurn > 1) {
```

Comment:
This code block is specific to game mechanics, not relevant in this library.

Explanation: 

LibTurnBasedGame should be able to serve variety of different games requiring turn-based functionality, but this if statement implies some specific game constraints. Should be implemented one abstraction layer above.